gems/faraday/2.5/faraday.rbs

```diff
@@ -186,6 +186,7 @@ module Faraday
     def headers=: (untyped) -> void
     def body=: (untyped) -> void
     def options=: (untyped) -> void
+    def headers: () -> Hash[untyped, untyped]
```

Comment:
According to [the YARD doc](https://github.com/lostisland/faraday/blob/18524c6e89071fbaf0592790f0f1da7a64107959/lib/faraday/request.rb#L22), this seems to return `Utils::Headers`. As far as I see the implementation in Faraday, an instance of `Utils::Headers` is assigned to `req.headers=` [here](https://github.com/lostisland/faraday/blob/18524c6e89071fbaf0592790f0f1da7a64107959/lib/faraday/connection.rb#L456); `headers.dup` comes from [`@headers`](https://github.com/lostisland/faraday/blob/18524c6e89071fbaf0592790f0f1da7a64107959/lib/faraday/connection.rb#L72). How about changing it like this?

```suggestion
    def headers: () -> Utils::Headers
```

Reply:
@yykamei
 
Thank you! I agree, so I've made the changes accordingly. Could you please review it again?
https://github.com/ruby/gem_rbs_collection/pull/628/commits/040c09e428a45e70f8973d9de9674b8527607ae5